### PR TITLE
clean code

### PR DESCRIPTION
**Pull request by samaradel**

### CODE REVIEW

File: .files/js/controllers/Machines/MachineEditController.js [hunk 1]
```diff
@@ -309,10 +309,10 @@
...
         Machine.delete($scope.machine.id).then(function() {
           LoadingDialog.hide();
-          var machine = _.find($scope.machines, function(machine) {
-            return machine.id === $scope.machine.id;
-            $scope.machines.splice($scope.machines.indexOf(machine) , 1);
-          });
+          // var machine = _.find($scope.machines, function(machine) {
+          //   $scope.machines.splice($scope.machines.indexOf(machine) , 1);
+          //   return machine.id === $scope.machine.id;
+          // });
```
Comment (grimpy) [inline .files/js/controllers/Machines/MachineEditController.js]: How does removing this code still make the machine dissapear from the machine list?

File: .files/js/general.js [hunk 2]
```diff
@@ -20,10 +20,10 @@ $('.dropdown-accordion').on('click', 'a[data-toggle="collapse"]', function(event
   $(this).parents('.panel-heading').toggleClass('open');
 });
 
-  // (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  // (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  // m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  // })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-  // ga('create', 'UA-47253978-1', 'mothership1.com');
-  // ga('send', 'pageview');
+// (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (grimpy) [inline .files/js/general.js]: just remove this block

File: .files/js/controllers/CloudSpaceNavigatorController.js [hunk 3]
```diff
@@ -70,32 +71,32 @@
...
-                id: account.id,
-                name: account.name,
-                cloudspaces: [],
-                acl: account.acl,
-              };
-              accountCloudSpaceHierarchy.push(hierarchyaccount);
-          }
-      }
-      $scope.noAccount = accountCloudSpaceHierarchy.length == 0;
+      // for (var idx in $scope.accounts) {
```
Comment (samaradel) [inline .files/js/controllers/CloudSpaceNavigatorController.js]: modified

File: .files/js/controllers/Machines/MachineEditController.js (hunk 1 above)
Comment (samaradel) [inline .files/js/controllers/Machines/MachineEditController.js]: modified

File: .files/js/general.js (hunk 2 above)
Comment (samaradel) [inline .files/js/general.js]: well..

**Pull request merged**